src/utilities/Libify.js

```diff
@@ -402,7 +402,49 @@ Libify.buildTransaction = function(attributes, operations, networkPassphrase) {
   } catch(e) {
     result.errors.push(e.message);
   }
+  return result;
+}
+
+Libify.buildFeeBumpTransaction = function(attributes, networkPassphrase) {
+  const result = {
+    errors: [],
+    xdr: '',
+  }
+
+  const { innerTxXDR, maxFee, sourceAccount } = attributes;
+  let innerTx;
+  try {
+    innerTx = new Sdk.TransactionBuilder.fromXDR(innerTxXDR, networkPassphrase);
+  } catch(e) {
+    result.errors.push('Invalid inner transaction XDR.');
+    return result;
+  }
```

Comment:
custom error here because easy to make XDR paste mistake (eg. missing a char) and ugly JS error shows

Reply:
@abuiles possible improvement to be made to the SDK?